service/src/main/java/com/ge/predix/acs/config/RedisConfig.java

```diff
@@ -43,11 +45,33 @@
     private String redisHost;
     @Value("${REDIS_PORT:6379}")
     private int redisPort;
+    @Value("${REDIS_MIN_ACTIVE:0}")
+    private int minActive;
+    @Value("${REDIS_MAX_ACTIVE:100}")
+    private int maxActive;
+    @Value("${REDIS_MAX_WAIT_TIME:30000}")
```

Comment:
I checked the REDIS transaction times in New Relic and found that they are sub-milliseconds. So it is safe to reduce the REDIS to 2 seconds. 30 seconds would be too long.